scripts/insert_into_appointments.sql

```diff
@@ -1 +1,17 @@
 -- insert appointments creating connections between doctors and patients
+
+INSERT INTO appointments(date, time, comment, doctor_id, patient_id)
+  -- Insert 4 appointments
+
+  -- Marsha Wilcox seeing Dr. Vicky Hilton at 10am on August 12, 2016.
+  VALUES ('2016-08-12', '10:00 am', '', 5, 9),
+
+-- Ivory Talley seeing Dr. Spencer Hubbard at 1pm on August 17, 2016.
+          ('2016-08-17', '1:00 pm', '', 7, 6),
+
+-- Marsha Wilcox seeing Dr. Spencer Hubbard at 10am on August 12, 2016.
+          ('2016-08-12', '10:00 am', '', 7, 6),
+
+-- Blake Hodge seeing Dr. Vicky Hilton at 1pm on August 17, 2016.
+          ('2016-08-17', '1:00 pm', '', 5, 8),
+;
```

Comment:
Hard-coding values like this will work, so I won't take points off, but it probably would have been a better idea to look up the specific patients and doctors from their respective tables. See Antony's solution branch for an example.